src/planner/linux.rs

```diff
@@ -1,21 +1,31 @@
+use std::{collections::HashMap, path::Path};
+
+use tokio::process::Command;
+use which::which;
+
+use super::ShellProfileLocations;
+#[cfg(all(target_os = "macos", not(feature = "nix-community")))]
+use crate::action::common::ConfigureDeterminateNixdInitService;
 use crate::{
     action::{
         base::{CreateDirectory, RemoveDirectory},
-        common::{ConfigureInitService, ConfigureNix, CreateUsersAndGroups, ProvisionNix},
-        linux::ProvisionSelinux,
+        common::{
+            ConfigureNix, ConfigureUpstreamInitService,
+            CreateUsersAndGroups, ProvisionDeterminateNixd, ProvisionNix,
```

Comment:
I did this, but I will note that it did make me wonder whether we should just revert the change that extracted 
`use crate::action::common::ConfigureDeterminateNixdInitService;` from this block in order to apply `#[cfg(not(feature = "nix-community"))]` to it (and the exclusion down around 270ish) so that we don't have a formatting diff?

When we discussed this, we decided it was net-good to have these since they helped ~document the intent/effect of the change to snip out the `--determinate` flag. Maybe it'd be better to just put comments on them?